Emby.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -416,27 +275,9 @@ public class DeviceManagerEntryPoint : IServerEntryPoint
             _logger = logger;
         }
 
-        public async Task RunAsync()
+        public Task RunAsync()
```

Comment:
Since this does nothing, is it safe to remove the parent class `DeviceManagerEntryPoint` ?

Reply:
I think so. However I would really like this in a separate PR if possible, to limit the scope of single PR to an atomic set of changes. However, seeing you've already dropped other instances of camera upload, we might as well finish it here...